lib/models/get_repositories/repos_result_rto.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import 'repository_rto.dart';
+
+part 'repos_result_rto.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class ReposResultRTO {
```

Comment:
What is a RTO? Why us acronyms?

Reply:
RTO -> RemoteTransferObject
To understand if it's a model object from the server - Any suggestions on this?

Reply:
As per the discussion changed all the objects with suffix RTO replaced with **Response**